File: Beecrowd/Python/bee1134.py
answer = 1
alcohol = 0
gas = 0
diesel = 0

while answer != "4":
    answer = input()

    match answer:
        case "1":
            alcohol += 1
        case "2":
            gas += 1
        case "3":
            diesel += 1

print(f"MUITO OBRIGADO\nAlcool: {alcohol}\nGasolina: {gas}\nDiesel: {diesel}")

File: Beecrowd/C/bee1134.c
#include <stdio.h>

int main() {

  int usr_input, alcohol = 0, gasoline = 0, diesel = 0;

  while (1) {
    scanf("%d", &usr_input);

    if (usr_input == 1)
      alcohol += 1;
    else if (usr_input == 2)
      gasoline += 1;
    else if (usr_input == 3)
      diesel += 1;
    else if (usr_input == 4)
      break;
  }

  printf("MUITO OBRIGADO\n");
  printf("Alcool: %d\n", alcohol);
  printf("Gasolina: %d\n", gasoline);
  printf("Diesel: %d\n", diesel);

  return 0;
}
